lib/polly_stubs.c

```
@@ -35,19 +35,25 @@ CAMLprim value caml_polly_create1(value val_unit)
 	CAMLreturn(val_res);
 }
 
+/* necessary because of changes from 4.X to 5.X in ocaml,
```

I think this can be cut as we can raise any exception we like from OCaml.

---

I was meaning that uerror is a macro in 0Caml 5 and caml_unix_error (or something like that is a new function). 
We have to define a function if we want external to work. My comment is not very clear, I update it.